includes/admin/class-sensei-learner-management.php

```diff
@@ -193,8 +193,8 @@ public function enqueue_scripts() {
 	 */
 	public function enqueue_styles() {
 
-		wp_enqueue_style( 'woothemes-sensei-admin' );
-		wp_enqueue_style( 'woothemes-sensei-jquery-ui', Sensei()->plugin_url . 'assets/css/jquery-ui.css', '', Sensei()->version );
+		wp_enqueue_style( 'sensei-admin' );
```

Comment:
This may be beyond the scope of this PR, but I do not see any style registered with this name.

Reply:
Good catch! Removed in ff867b018f67448905d9d6cdb4fc7a020aca57e8

Reply:
👍